tagstudio/src/qt/widgets/thumb_renderer.py

```diff
@@ -289,8 +286,8 @@ def render(
                     math.ceil(adj_size / pixel_ratio),
                     math.ceil(final.size[1] / pixel_ratio),
                 ),
-                extension,
+                _filepath.suffix,
             )
 
         else:
-            self.updated.emit(timestamp, QPixmap(), QSize(*base_size), extension)
+            self.updated.emit(timestamp, QPixmap(), QSize(*base_size), _filepath.suffix)
```

Comment:
```suggestion
            self.updated.emit(timestamp, QPixmap(), QSize(*base_size), _filepath.suffix[1:])
```
Removes leading "." from filetype display on thumbnails